### ISSUE TITLE
Unable to import library

### ISSUE THREAD

**Issue by kcprophecy** [evilhero/mylar#1568]
Hello I am trying to import my library into mylar and am running into the following error:

2017-02-11 21:51:50	ERROR	[IMPORT] Unable to complete the scan: [Errno 2] No such file or directory
2017-02-11 21:51:50	DEBUG	Extracting primary image savage_sword_84_01.jpg as coverfile for display.
2017-02-11 21:51:50	DEBUG	CBZ file detected. Checking for .xml within file
2017-02-11 21:51:50	INFO	[IMPORT-CBZ} Attempting to read tags present in filename: /mnt/storage/media/library/Comics/The Savage Sword of Conan (1974)/The Savage Sword of Conan 84 (1983).cbz
2017-02-11 21:51:50	DEBUG	[IMPORT-CBZ] Metatagging checking enabled.
2017-02-11 21:51:50	DEBUG	Analyzing : The Savage Sword of Conan 84 (1983).cbz

------------------------------------------------------------------------------------------------------------

Not sure why this error is popping up. I opened the cbz file with winrar and the referenced jpg is there. I have also attempted to move this entire comic series out and re-import but then it just gets stuck on a new series with the same error. Any ideas?

**Comment by evilhero** [comment]
It might be the paths you've specified for locations. Mylar needs to have full access to the given directory to import. When it reads the tags, it actually copies the file to the mylar cache folder, extracts it to check the metadata and then deletes it. So I'm guessing what's happening is the there might be a permissions issue of some kind, either in the cache or in the directory you're trying to import from?

**Comment by kcprophecy** [comment]
Hey evilhero, thanks for the quick response back. Permissions are definitely a little different than my previous setup but still no dice even setting both the Comics and cache directory/files to 777. Is it possible to move the Mylar db from one setup to another?

**Comment by evilhero** [comment]
You can move it but the paths that mylar expects series to be located in might be wrong if it's on a different system (mylar stores the absolute paths to each series in the dB file). 

Are you trying to import from an external drive or something similar by chance?

**Comment by kcprophecy** [comment]
Comics were located on a nfs share previously. I moved mylar to an unprivileged container though and also moved over all my data from one server to another changing paths along the way. I followed the directions that you posted previously here, https://forum.mylarcomics.com/viewtopic.php?f=12&t=601&sid=bea95e783df1cd3baa9daeab1a2002c2, setting a symbolic link from new path to old. Looks like it worked? Post processing went well so I suppose everything is back how it was. Thanks man.
